<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdicionarClientes</class>
 <widget class="QDialog" name="AdicionarClientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>598</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>860</width>
    <height>598</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Clientes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(2, 0, 36, 255), stop:1 rgba(239, 77, 18, 255));
font-family: Courier;
}

QPushButton{
  font-size:15px;
  border-width:1px;
  border-color:#1f2f47;
  border-radius:6px;
  height:25px
}

QPushButton{background:qlineargradient(spread:reflect, x1:0.994318, y1:0.915, x2:0.994318, y2:0.488, stop:0.1875 rgba(128, 128, 128, 255), stop:1 rgba(211, 211, 211, 255))}

QPushButton::pressed
{
	background:qlineargradient(spread:reflect, x1:0.994318, y1:0.915, x2:0.994318, y2:0.488, stop:0.1875 rgba(227, 156, 41, 255), stop:1 rgba(211, 211, 211, 255));
margin-top:1px;
margin-bottom:-1px;

}
QPushButton::hover
{
	background:qlineargradient(spread:reflect, x1:0.994318, y1:0.915, x2:0.994318, y2:0.488, stop:0.1875 rgba(227, 156, 41, 255), stop:1 rgba(211, 211, 211, 255));
}

QComboBox{
  border-color:#1f2f47;
  border-radius:4px
}

QLineEdit{
  height:22px;
  border-width:1px;
  border-radius:4px;
  border-color:#fff;
  background:#b5b5b5
}

QTextEdit{
  background:#b5b5b5;
  border:6px;
}

#twCliente
{	
	 selection-background-color: rgb(250, 128, 114);
	 background:#b5b5b5;
	 border-bottom-right-radius:20px;
     border-bottom-left-radius:20px;
     border: 1px solid #F4A460;
}

</string>
  </property>
  <widget class="QTabWidget" name="tabCliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabIncluirCliente">
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <attribute name="title">
     <string>Incluir</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonIncluir">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>501</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
width: 120px;

}</string>
     </property>
     <property name="text">
      <string>Incluir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background:#ff9318;
	border-top-right-radius:50px;
	border-bottom-left-radius:50px;
	font-family: Courier;
}</string>
     </property>
     <property name="text">
      <string>               Informações do Cliente</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>501</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditNomeIncluir">
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Nome do Cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEnderecoIncluir">
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Endereço (Opcional)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTelefoneIncluir">
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Telefone (Opcional)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Email (Opcional)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabListarCliente">
    <attribute name="title">
     <string>Lista de Clientes</string>
    </attribute>
    <widget class="QTableWidget" name="twCliente">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>501</width>
       <height>471</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> selection-background-color: rgb(250, 128, 114);</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSearchCliente">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="placeholderText">
      <string>Buscar Cliente</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxOrdem">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>490</y>
       <width>164</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Text</family>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
	background:qlineargradient(spread:reflect, x1:0.994318, y1:0.915, x2:0.994318, y2:0.488, stop:0.1875 rgba(128, 128, 128, 255), stop:1 rgba(211, 211, 211, 255));

width:146;
height:30

}

</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Crescente por nome</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Decrescente por nome</string>
      </property>
     </item>
    </widget>
    <widget class="QFrame" name="fr_DadosCliente">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>261</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonExcluir">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>224</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0;
border-bottom: 1px solid #F4A460;
margin: 14px 0;


</string>
           </property>
           <property name="text">
            <string>Nome    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditNomeEdit">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string>Nome do Cliente</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0;
border-bottom: 1px solid #F4A460;
margin: 14px 0;
</string>
           </property>
           <property name="text">
            <string>Endereço</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEnderecoEdit">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>Endereço</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0;
border-bottom: 1px solid #F4A460;
margin: 14px 0;
</string>
           </property>
           <property name="text">
            <string>Telefone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTelefoneEdit">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>Telefone</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0;
border-bottom: 1px solid #F4A460;
margin: 14px 0;
</string>
           </property>
           <property name="text">
            <string>Email   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEmailEdit">
           <property name="font">
            <font>
             <family>Courier</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>Email</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="quadradim" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>251</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #F4A460;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelID">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>500</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <zorder>quadradim</zorder>
    <zorder>twCliente</zorder>
    <zorder>lineEditSearchCliente</zorder>
    <zorder>comboBoxOrdem</zorder>
    <zorder>fr_DadosCliente</zorder>
    <zorder>labelID</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
